### ISSUE TITLE
multibyte string error

### ISSUE THREAD

**Issue by omikuji** [bolcom/libunftp#133]
Hello! I facing a problem.

Success case

```
Command:	PASV
Response:	227 Entering Passive Mode (127,0,0,1,234,79)
Command:	STOR a
Response:	150 Ready to receive data
Response:	226 File successfully written
Status:	File transfer successful, transferred 0 B in 1 second
```

Error case

```
Command:	PASV
Response:	227 Entering Passive Mode (127,0,0,1,203,51)
Command:	STOR あ
Response:	500 Invalid UTF8 in command
Error:	Critical file transfer error
```

Error from here
https://github.com/bolcom/libunftp/blob/master/src/server/commands/mod.rs#L593-L595

How are you working it out?

**Comment by hannesdejager** [comment]
I didn't notice this issue until now. Thanks for logging it.

**Comment by hannesdejager** [comment]
This should be fixed in libunftp 0.14.1
